server/config/index.js

```diff
@@ -0,0 +1,26 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const {
+  NODE_ENV, DB_URL_DEV, DB_URL_TEST, DB_URL_PROD
+} = process.env;
+
+const dbUrl = {
+  development: {
+    url: DB_URL_DEV
+  },
+  test: {
+    url: DB_URL_TEST
+  },
+  production: {
+    url: DB_URL_PROD
+  }
+};
+
+const connectionString = dbUrl[NODE_ENV].url;
+
+const pool = new Pool({ connectionString });
+
+export default pool;
```

Comment:
Awesome!! This logic is good, instead of using an if or a switch statement this is preferable. Good job.

Reply:
Thank you Nero

Reply:
This is really cool @LukasChiama Some JavaScript ninja kung fu going on here. lol! I like this, it's a very elegant implementation!